HW-3/task2.py

```diff
@@ -0,0 +1,12 @@
+# Дан список повторяющихся элементов. Вернуть список
+# с дублирующимися элементами. В результирующем списке
+# не должно быть дубликатов
+
+my_list = [4, 8, True, 'dff', 8, 4, True, 'fff', 7, 2, 8]
+tmp = []
+for item in my_list:
```

Comment:
Высокая сложность, чтобы ее немного уменьшить, можно сделать так.
```suggestion
for item in set(my_list):
```